app/src/main/res/values/strings.xml

```diff
@@ -79,174 +79,174 @@
     <string name="grid_size">Grid size</string>
     <string name="search_on_s">Search on %s</string>
     <string name="delete_manga">Delete manga</string>
-    <string name="text_delete_local_manga">Do you really want to delete \"%s\" from your phone\'s local storage? \nThis operation cannot be undone.</string>
+    <string name="text_delete_local_manga">Delete \"%s\" from device permanently?</string>
     <string name="reader_settings">Reader settings</string>
     <string name="switch_pages">Switch pages</string>
-    <string name="taps_on_edges">Taps on edges</string>
+    <string name="taps_on_edges">Edge taps</string>
     <string name="volume_buttons">Volume buttons</string>
     <string name="_continue">Continue</string>
     <string name="warning">Warning</string>
-    <string name="network_consumption_warning">This operation may consume a lot of network traffic</string>
+    <string name="network_consumption_warning">This may download a lot of power</string>
```

Comment:
The meaning is missed

Reply:
I have no idea what I was thinking here. Good catch.